db/getItemsOnList_test.go

```diff
@@ -22,6 +23,14 @@ func (m *mockDB) Query(input *dynamodb.QueryInput) (*dynamodb.QueryOutput, error
 	return args.Get(0).(*dynamodb.QueryOutput), args.Error(1)
 }
 
+var testConfig config.Config = config.Config{
```

Comment:
Could declare this in the start of `TestGetItemsOnList` to keep its scope smaller.
Unless you want it visible outside `TestGetItemsOnList`. idm